app/views/languages/repository.erb

```diff
@@ -0,0 +1,79 @@
+  <section class="contributing-intro jumbotron">
+    <h1>Exercism Repositories</h1>
+
+    <p class="lead">
+      <strong>What are the repositories you can contribute to?</strong><br />
+    </p>
+
+    <p> Exercism is a project comprised of several repositories and these can be divided basically into:</p>
+
+      <h2> Basic components </h2>
+    <p>
+      <ul>
+          <li><strong><a href="https://github.com/exercism/exercism.io">The website</a></strong>: Holds the code that people submit. Browse and review solutions, and have conversations.</li>
+          <li><strong><a href="https://github.com/exercism/cli">Command line interface</a></strong>: Used to fetch exercises and submit solutions.</li>
+           <li><strong><a href="https://github.com/exercism/x-api">Problems API</a></strong>: Delivers exercises. Consumed by the Exercism command-line client, as well as the Exercism website.</li>
+      </ul>
+    </p>
+
+        <h2> Interesting Tools </h2>
+
+
+    <h2> Basic components </h2>
+
+     <ul>
+      <li><strong><a href="https://github.com/exercism/exercism.io">The website</a></strong>: Holds the code that people submit. Browse and review solutions, and have conversations.</li>
+      <li><strong><a href="https://github.com/exercism/cli">Command line interface</a></strong>: Used to fetch exercises and submit solutions.</li>
+      <li><strong><a href="https://github.com/exercism/x-api">Problems API</a></strong>: Delivers exercises. Consumed by the Exercism command-line client, as well as the Exercism website.</li>
+     </ul>
+
+     <h2> Interesting Tools </h2>
+
+
+    <p>
+        <ul>
+          <li><strong><a href="https://github.com/exercism/rikki">Rikki- the robot</a></strong>: Provides helpful, friendly, automated feedback on certain exercises.</li>
+          <li><strong><a href="https://github.com/exercism/blazon">Blazon</a></strong>: Helps us manage cross-track issues.</li>
+        </ul>
+
+    </p>
+  </section>
+
+      <section class="page-header">
+        <h3>Languages you can contribute to!</h3>
+      </section>
+
+
+      <% active.each_slice(6) do |tracks| %>
+        <div class="row">
+          <% tracks.each do |t| %>
+            <%= erb :'languages/_repo', locals: { track: t } %>
+          <% end %>
+        </div>
+      <% end %>
+
+      <section class="page-header">
+        <h3>Upcoming</h3>
+      <p>Help contribute to new languages to exercism.io by adding more exercises, and helping to review code. To see what still needs to be done, view the launch checklist.</p>
+      </section>
+          <% upcoming.each_slice(6) do |tracks| %>
+        <div class="row">
+          <% tracks.each do |t| %>
+            <%= erb :'languages/_repo', locals: { track: t } %>
+          <% end %>
+        </div>
+      <% end %>
+
+      <section class="page-header">
+        <h3>Planned</h3>
+      <p>Requested languages that need some tender loving care. Pitch in by completing the checklist.</p>
```

Comment:
I think it would be useful to note how _upcoming_ and _planned_ differ.

In upcoming tracks, people have been contributing exercises.

Planned tracks have been requested, but don't have much in the way of exercises yet.